src/functionaltests/java/com/ericsson/ei/subscriptions/query/QueryAggregatedObjectsTestSteps.java

```diff
@@ -0,0 +1,232 @@
+
+        } catch (IOException e) {
+            LOGGER.error("Failed to open test json files for test. Error message\n: " + e.getMessage());
+            e.printStackTrace();
+        }
+    }
+    
+    @Given("^Aggregated object is created$")
+    public void aggregated_object_is_created() throws Throwable {
+        LOGGER.debug("Creating aggregated object in MongoDb");
+        assertEquals(true, createDocumentInMongoDb(eiDatabaseName, aggrCollectionName, aggrObj));
+    }
+
+    @Given("^Missed Notification object is created$")
+    public void missed_notification_object_is_created() throws Throwable {
+        LOGGER.debug("Missed Notification object has been created in MongoDb");
+        assertEquals(true, createDocumentInMongoDb(missedNotificationDatabaseName, missedNotificationCollectionName, missedNotificationObj));
+    }
+
+    @Then("^Perform valid query on newly created Aggregated object")
+    public void perform_valid_query_on_newly_created_aggregated_object() throws Throwable {
+        final String expectedTestCaseFinishedEventId = "cb9d64b0-a6e9-4419-8b5d-a650c27c1111";
+        final String entryPoint = "/queryAggregatedObject";
+        final String documentId = "6acc3c87-75e0-4b6d-88f5-b1a5d4e62b43";
+        LOGGER.debug("Got AggregateObject actual DocumentId after querying MongoDB: " + documentId);
+        mvcResult = mockMvc.perform(get(entryPoint)
+                .param("ID", documentId)
+                .accept(MediaType.APPLICATION_JSON)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andReturn();
+        LOGGER.debug("Response of /queryAggregatedObject RestApi, Status Code: " + mvcResult.getResponse().getStatus() +
+                           "\nResponse: " + mvcResult.getResponse().getContentAsString());
+        
+        JsonNode jsonNodeResult = objMapper.readValue(mvcResult.getResponse().getContentAsString(), JsonNode.class);
+        JsonNode responseEntityNode = objMapper.readValue(jsonNodeResult.get("responseEntity").asText(), JsonNode.class); 
+        String responseEntityFormattedString = responseEntityNode.toString().
+                substring(1, responseEntityNode.toString().length()-1);
+        
+        LOGGER.debug("AggregatedObject from Response: " + responseEntityFormattedString);
+        JsonNode responseEntityFormattedJsonNode = objMapper.readValue(responseEntityFormattedString, JsonNode.class);
+        String actualTestCaseFinishedEventId = responseEntityFormattedJsonNode.get("aggregatedObject")
+                .get("testCaseExecutions").get(0).get("testCaseFinishedEventId").asText();
+        
+        assertEquals(HttpStatus.OK.toString(), Integer.toString(mvcResult.getResponse().getStatus()));
+        assertEquals("Failed to compare actual Aggregated Object:\n" + expectedTestCaseFinishedEventId
+                + "\nwith expected Aggregated Object:\n" + actualTestCaseFinishedEventId,
```

Comment:
The expectedTestCaseFinishedEventId and actualTestCaseFinishedEventId shopuld switch places in the DEBUG message.